akka/src/test/scala/better/files/FileWatcherSpec.scala

```diff
@@ -26,10 +26,10 @@ class FileWatcherSpec extends FlatSpec with Matchers {
     }
     /***************************************************************************/
     import java.nio.file.{StandardWatchEventKinds => Events}
-    import FileWatcher._
 
-    import akka.actor.{ActorRef, ActorSystem}
-    implicit val system = ActorSystem()
+    import ActorSystemSupport.system
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/pathikrit/better-files/file/3977902099/issues/source?bid=2860326&fileBranchId=3676352#l30)